### PR TITLE
fix: cannot find user home folder errror

### LINKED ISSUE #104: Registry configuration fails on Windows: `Unable to find home directory`
image

### PR DESCRIPTION
**Pull request by dgolovin**
Fix #104.

### CODE REVIEW

**Review by benoitf — COMMENTED** [review]
should it use os.homedir() ?

**Review by vrothberg — COMMENTED** [review]
Looking at the code, I think it's much safer to use `podman login --get-login registry.redhat.io`.  This way, we'll stick to how `podman login` behaves in the specific environment.

If the exit code is 0, the user is logged in.

**Comment by dgolovin** [comment]
@vrothberg A fix is better than refactoring at this point.

File: src/extension.ts [hunk 1]
```diff
@@ -101,7 +102,7 @@ function removeRegistry(serverUrl: string = REGISTRY_REDHAT_IO): void {
 // TODO: add listRegistries to registry API to allow search by
 // registry URL
 function isRedHatRegistryConfigured(): boolean {
-  const homeFolderPath = process.env.HOME ? process.env.HOME : process.env.USER_PROFILE;
+  const homeFolderPath = homedir();
```
Comment (benoitf) [inline src/extension.ts:105]: looks like it's always returning something so i'm not sure about the next condition if not defined

**Review by benoitf — APPROVED** [review]

File: src/extension.ts (hunk 1 above)
Comment (dgolovin) [reply src/extension.ts]: removed check for home dir not being set.

**Pull request merged**